web/src/compose_notifications.ts

```diff
@@ -50,9 +52,21 @@ export function notify_above_composebox(
             link_text,
         }),
     );
-    // We pass in include_unmute_banner as false because we don't want to
-    // clear any unmute_banner associated with this same message.
-    compose_banner.clear_message_sent_banners(false);
```

Comment:
Why is it correct to remove `compose_banner.clear_message_sent_banners` here?

Reply:
`compose_banner.clear_message_sent_banners` method has to be called only once for every message

In some cases, there are multiple message sent banners, thus only one banner was showing at a time due to calling it twice. Moving it outside would the function solved this issue.

One case is when message is outside view:

![image](https://github.com/zulip/zulip/assets/78212328/ec4b1194-9971-4047-913f-53a64469b482)

If we make the banner not a message sent banner, it just stacks up.